src/Styleguide/Pages/Artwork/Sidebar/Artists.tsx

```diff
@@ -14,18 +14,16 @@ export interface ArtistsProps {
   }>
 }
 
-const ArtistsContainer = styled.div.attrs<SpaceProps>({})`
-  ${space};
-`
+const ArtistsContainer = styled(Box)``
```

Comment:
You actually don't need to do `styled(Box)`` `. You can just use `ArtistsContainer = Box` being that `Box` is already a styled component.